### DISCUSSION TITLE
Build Issue

### DISCUSSION THREAD

**Discussion by roams04** [YannickRe/spfx-msgraph-peoplesearch#84] [Q&A]
Please help me to figure out what I am doing wrong.

Downloaded version 2.8.0.0
use node 18.19.1
ran npm install
when I ran gulp build I got error
Error - [tsc] src/services/TemplateService/TemplateService.ts(136,17): error TS2322: Type 'number | IComponentFieldsConfiguration[]' is not assignable to type 'any[]'.
[tsc] Type 'number' is not assignable to type 'any[]'.
Error - 'tsc' sub task errored after 17 s exited with code 2
'build' errored after 19 s

Thank you

**Comment by YannickRe** [comment]
This is fixed in v2.8.4. In the future, please create an issue for this as that's way easier to track.
